students/Dustin_L/lesson07/html_render.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+"""HTML Render Module
+
+This module contains all of the functions for the HTML Render module
+"""
+
+
+class Element(object):
```

Comment:
In Python3 it's not necessary to explicitly subclass from `object`. Base classes are conventionally defined like this:
```
class Element:
```